lua/ulx/modules/sh/cfc_owoify.lua

```diff
@@ -0,0 +1,44 @@
+local CATEGORY_TYPE = "Chat"
+local OWOIFYDICT = { ove = "uv", na = "nya", ne = "nye", ni = "nyi", no = "nyo", nu = "nyu", r = "w", l = "w", O = "owo", U = "uwu", wh = "w" }
+local owoifyTargets = {}
+
+local function owoifyString( inputStr )
+    for k, v in pairs( OWOIFYDICT ) do
+        inputStr = string.Replace( inputStr, k, v )
+    end
+
+    return inputStr
+end
+
+local function onPlayerSay( ply, inputStr )
+    if not table.HasValue( owoifyTargets, ply ) then return end
+
+    return owoifyString( inputStr )
+end
+
+local function owoifyOn( caller, targets )
+    table.Add( owoifyTargets, targets )
+    hook.Add( "PlayerSay", "CFC_ULX_OwoifyString", onPlayerSay )
+    ulx.fancyLogAdmin( caller, "#A owoified #T", targets )
+end
+
+local function owoifyOff( caller, targets )
+    for k, v in pairs( owoifyTargets ) do
+        owoifyTargets[k] = nil
+    end
+
+    if table.Count( owoifyTargets ) == 0 then
+        hook.Remove( "PlayerSay", "CFC_ULX_OwoifyString" )
+    end
```

Comment:
This is an efficiency nitpick here that has no actual bearing on your code:

I believe `table.Count` loops through all of the items in the table and increments a counter as it goes through. For small tables this is completely fine, but if you had a larger list you might want to keep track of the current count yourself. You control when items are added and removed, so you could manually modify a `owoifyCount` variable each time you do.

No need to change it here, just wanted to point that out.